sklego/mixture/bayesian_gmm_classifier.py

```diff
@@ -22,6 +22,48 @@ class BayesianGMMClassifier(BaseEstimator, ClassifierMixin):
         A dictionary of Bayesian Gaussian Mixture Models, one for each class.
     classes_ : np.ndarray of shape (n_classes,)
         The classes seen during `fit`.
+
+      Examples
+    --------
```

Comment:
Indentation should be:

```suggestion
    Examples
    --------
```